core/src/test/java/io/questdb/griffin/engine/functions/groupby/AvgDoubleVecGroupByFunctionFactoryTest.java

```diff
@@ -31,6 +31,9 @@
 
     @Test
     public void testAddColumn() throws Exception {
+        // fix page frame size, because it affects AVG accuracy
+        pageFrameMaxRows = 10_000;
```

Comment:
nit: why do we need to change the default here? Both the default (`10000000`) and this value are larger than the number of rows in the table under test. The same consideration applies to other tests here.

Reply:
grouping of rows affects double sum avg() accuracy